Entities/AHuman.cpp

```diff
@@ -4262,7 +4262,7 @@ void AHuman::Update()
         }
     }
     // Keel over
-    else if (m_Status == UNSTABLE)
+    else if (m_Status == UNSTABLE && fabs(m_AngularVel) < 5)
```

Comment:
This part mainly annoyed me when I was testing things on an actor that was pinned - once unstable conditions were met, it would infinitely gain angular vel (probably due to rot angle not updating) until reaching TooFast conditions. I guess it doesn't matter in actual gameplay, but I wanted to include it anyway.

Reply:
Ahhhhh, that makes sense. I was trying to figure out what this was fixing but couldn't really tell. Good stuff!